app/models/mhv_account.rb

```diff
@@ -120,8 +127,12 @@ def create_mhv_account!
 
   def upgrade_mhv_account!
     if may_upgrade?
+      StatsD.increment("#{STATSD_ACCOUNT_UPGRADE_KEY}.total")
+
       client_response = mhv_ac_client.post_upgrade(params_for_upgrade)
       if client_response[:status] == 'success'
+        StatsD.increment("#{STATSD_ACCOUNT_UPGRADE_KEY}.success")
+
```

Comment:
Check the test coverage. It was one of the tests and should have a VCR cassette. It should also be handled I think in the rescue block maybe if IRRC